core/modules/user/user.module

```diff
@@ -1744,6 +1779,11 @@ function user_login($form, &$form_state) {
     '#required' => TRUE,
     '#attributes' => array(
       'autofocus' => 'autofocus',
+      // Add attributes to field to prevent spell-jacking and unwanted
+      // automatic changes caused by the browser.
+      'autocapitalize' => 'none',
+      'autocorrect' => 'off',
+      'spellcheck' => 'false',
```

Comment:
This block simply adds the desired attributes to both name and password for this specific high-level form.